### DISCUSSION TITLE
OrchardCms Vote

### DISCUSSION THREAD

**Discussion by MichaelGaoX** [OrchardCMS/OrchardCore#15835] [Q&A]
How to add Vote,like I publish some isues,and anyone can vote what you chose that loading my website

**Comment by sebastienros** [comment]
There is no such thing right now, but ideally we could use a Voting module like the same I made for O1 (should be on the gallery). It's just an API to add votes to any content item. Then a custom part like RatingsPart that would use this API, and some Display modes to let users enter rating (5 stars, Like buttons, ...)

**Comment by MichaelGaoX** [comment]
@sebastienros  I think I need to learn about O1, just use OCMS in the first time.
